test/contracts/ForceMove/respondWithAlternative.test.ts

```diff
@@ -11,10 +11,10 @@ import {
   signStates,
   sendTransaction,
 } from '../../test-helpers';
-import {AddressZero} from 'ethers/constants';
-import {Outcome} from '../../../src/contract/outcome';
+import {AddressZero, One, HashZero} from 'ethers/constants';
+import {Outcome, hashOutcome} from '../../../src/contract/outcome';
 import {Channel, getChannelId} from '../../../src/contract/channel';
-import {State} from '../../../src/contract/state';
+import {State, hashState} from '../../../src/contract/state';
 import {hashChannelStorage} from '../../../src/contract/channel-storage';
```

Comment:
The only test that fails here is due to not updating the implementation of `hashChannelStorage`.